docs/modules/ROOT/pages/index.adoc

```diff
@@ -964,13 +941,13 @@ Any content including, but not limited to the _<<_named_actor,named actor’s>>_
 
 The effectiveness of such attacks will necessarily be dependent on the language and other related development tooling in use for any given implementation. Beyond reminding implementers that parsing and validation errors are a likely attack surface, it is outside the scope of this specification to provide language-specific guidance.
 
-==== Homoglyph and typo-squatting attacks
+==== Name collisions and homoglyph and typo-squatting attacks
 
 An attacker could make use of visually-similar Unicode characters to mislead an end user into accepting a mistaken assertion of identity on behalf of a specific _<<_named_actor,named actor>>._ Such attacks are common in phishing and impersonation attacks conducted on domain names and social media.
 
-Similarly, it is likely that two or more _<<_named_actor,named actors>>_ may legitimately have the same or very similar names. This is discussed in xref:_name_collisions[xrefstyle=full].
+Similarly, it is likely that two or more _<<_named_actor,named actors>>_ may legitimately have the same or very similar names. Whether legitimate or not, similar names may cause confusion among _<<_identity_assertion_consumer,identity assertion consumers>>_ absent other signals that disambiguate specific individuals or organizations.
```

Comment:
Perhaps add a sentence:

"If anonymous identities are supported, _<<_identity_assertion_consumer,identity assertion consumers>> _would also require signals to disambiguate between anonymous identities."